include/spl/sys/kstat.h

```diff
@@ -69,6 +69,7 @@
 #define	KSTAT_FLAG_WRITABLE	0x04
 #define	KSTAT_FLAG_PERSISTENT	0x08
 #define	KSTAT_FLAG_DORMANT	0x10
+#define	KSTAT_FLAG_NO_HEADERS	0x20
```

Comment:
We should keep this aligned with the Illumos ones for consistency and add yours to the end.

```
#define KSTAT_FLAG_VIRTUAL              0x01
#define KSTAT_FLAG_VAR_SIZE             0x02
#define KSTAT_FLAG_WRITABLE             0x04
#define KSTAT_FLAG_PERSISTENT           0x08
#define KSTAT_FLAG_DORMANT              0x10
#define KSTAT_FLAG_INVALID              0x20
#define KSTAT_FLAG_LONGSTRINGS          0x40
```